bundler/lib/bundler/friendly_errors.rb

```diff
@@ -100,15 +100,15 @@ def request_issue_report_for(e)
         #{issues_url(e)}
 
         If there aren't any reports for this error yet, please create copy and paste the report template above into a new issue. Don't forget to anonymize any private data! The new issue form is located at:
-        https://github.com/rubygems/bundler/issues/new
+        https://github.com/rubygems/rubygems/issues/new?labels=Bundler
       EOS
     end
 
     def issues_url(exception)
       message = exception.message.lines.first.tr(":", " ").chomp
       message = message.split("-").first if exception.is_a?(Errno)
       require "cgi"
-      "https://github.com/rubygems/bundler/search?q=" \
+      "https://github.com/rubygems/rubygems/search?q=" \
```

Comment:
We could filter by the “Bundler” label but it might be better to just search for _all_ relevant issues (even if they haven’t been properly tagged yet).